azerothcore/azerothcore-wotlk
# fix(Core/Spell): Instant Statue

## Changes Proposed:
- Currently, the "pedestal" creature is not being removed if you remove the aura.
- One of the auras that Instant Statue Pedestal applies on the player is not removed (fixed with spell_linked_spell)

## Tests Performed:
- Builds
- Tested in-game



## How to Test the Changes:

1. .add 54212
2. use the item
3. Rightclick the aura to remove it
4. See that the pedestal is removed


## Target Branch(es):








 
## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
